### ISSUE TITLE
Instructor edit rubric question: ignore empty rows

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#7039]
v5.100

Steps:
1. Login as instructor.
2. Edit a rubric question. Leave an empty row in the middle. Try to save.

Actual: All edits are discarded and an error message is given
Suggestion: Either inform about the blank row without discarding all edits, or ignore the empty row and save the rest.

**Comment by VamsiSangam** [comment]
@damithc sir, currently removing an entire row and submitting shows this error -

![1](https://user-images.githubusercontent.com/11662001/27897231-8dc4f80a-623c-11e7-86e8-390ceed70dd6.JPG)

So if we just added a `required` attribute to the sub question's `textarea` tag, we can stop the instructor on clicking `Save changes`. When the instructor clicks `Save changes` automatically a pop-up appears as such -

![capture](https://user-images.githubusercontent.com/11662001/27897302-189aaac4-623d-11e7-9128-af4896bbf65b.JPG)

The way I understand it, the rubric descriptions are optional, we are only concerned about empty sub questions. The above fix puts a front-end check for non-empty sub questions. We already have a back end check, so shall I proceed with this fix?
